### ISSUE TITLE
Error: Unable to resolve actions. Repository not found : pullreminders/label-when-approved-action.

### ISSUE THREAD

**Issue by sugarandmagic** [abinoda/label-when-approved-action#77]
Hi!

Thanks for a great action! Today we have noticed the action cannot be located, have you seen this or do you have any suggestions? It was working great until today.

`Error: Unable to resolve actions. Repository not found : pullreminders/label-when-approved-action.`

Thanks

**Comment by gsingh1** [comment]
+1

**Comment by Rainfords** [comment]
+1

**Comment by Rainfords** [comment]
If you click the `Use Latest Version` button [here ](https://github.com/marketplace/actions/label-approved-pull-requests?version=1.0.5) it gives:

```
- name: Label approved pull requests
  uses: abinoda/label-when-approved-action@1.0.5
```

Is this now the correct repo to use? its different from the repo in the instructions on the page.

**Comment by yafishman** [comment]
> If you click the `Use Latest Version` button [here ](https://github.com/marketplace/actions/label-approved-pull-requests?version=1.0.5) it gives:
> 
> ```
> - name: Label approved pull requests
>   uses: abinoda/label-when-approved-action@1.0.5
> ```
> 
I changed my custom action to the above path and it's working for me now

**Comment by sugarandmagic** [comment]
Yes so did I but I'm reluctant to keep using it; this has caused a fair bit of disruption and there's been no response from the author :(

**Comment by michaeltintiuc** [comment]
Apparently version 1.0.7 from releases doesn't work either, changing to master worked
